CHANGELOG.rst

```diff
@@ -2,6 +2,11 @@
 Changelog
 =========
 
+
+v0.18.3 (unreleased)
+--------------------
+* Fix bug affecting GriddedForcing, where `station_idx` in the call to `nc_specs` was set to 1 instead of None. (#PR #501)
```

Comment:
```suggestion
v0.18.3 (unreleased)
--------------------
* Fix bug affecting GriddedForcing, where `station_idx` in the call to `nc_specs` was set to ``1`` instead of ``None``. (#PR #501)
```